docs/recipes/Module-Composition.md

```diff
@@ -0,0 +1,63 @@
+<!--ONE-DOCS-HIDE start-->
+[👈 Return to Overview](./README.md)
+<!--ONE-DOCS-HIDE end-->
+
+# Module Composition
+
+A key part of working with One App is module composition. Modules can be rendered inside one another by either using `ModuleRoute` or `RenderModule`.
+
+## `ModuleRoute`
+
+`ModuleRoute` from `holocron-module-route` allows modules to dynamically load other modules
+as a child route. This can be done on any module as routing is not limited to the route module:
+
+```jsx
+ParentModule.childRoutes = () => [
+  <ModuleRoute
+    path="childModule"
+    moduleName="ChildModule"
+    greeting="hello"
+  />,
+];
+```
+
+When using ModuleRoute, additional props are passed through to the module via the `route` prop:
+
+```jsx
+const ChildModule = ({ route: { greeting } } = {}) => <h1>{greeting}</h1>;
+```
+
+## `RenderModule`
+
+Holocron's `RenderModule` provides an alternate method to rendering another module:
+
+```jsx
+const ParentModule = () => (
+  <div>
+    <h1>I am the parent module</h1>
+    <RenderModule moduleName="ChildModule" props={{ greeting: 'hello' }} />;
+  </div>
+);
+```
+
+To use RenderModule we need to ensure that the module bundle has been correctly
+loaded into our client or server before it can be rendered. We can do this by
```

Comment:
All server bundles are already stored in memory. On the server it's only the module's data we would need to load.

Reply:
true true